modules/odc-grouping-multiple-components.adoc

```diff
@@ -15,23 +15,23 @@ The following procedure adds a MongoDB database service to an existing applicati
 
 .Procedure
 
-. Create and deploy the MongoDB service to your project as follows:
-.. In the *Developer* perspective, navigate to the *Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
-.. Click on the *MongoDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MongoDB service, and click *Create* to create the service.
+. Create and deploy the MariaDB service to your project as follows:
+.. In the *Developer* perspective, navigate to the *+Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
```

Comment:
```suggestion
.. In the *Developer* perspective, navigate to the *+Add* page and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
```

Reply:
When user clicks on Database card on +Add page, it will open the Developer Catalog page with Database options. In above sentence, it seems that you have added description of Developer Catalog page instead it should what user will see when he/she clicks on Database card on Add page. WDYT about updating above sentence? @Preeticp @rishumehra

Reply:
@gajanan-more the change from view to page, if done needs to be uniform across the OCP doc set. If required we will pick that change up through another PR.
Suggest similar wording as we have in the other module:
. In the *Developer* perspective, navigate to the *+Add* page and select the *Database* option to view all the available services in the *Developer Catalog*.
. Select *Databases* to list all the database services, and then click on the MariaDB option to see the details for the service.
Does this make it more clear?

Reply:
Option 1: 
. In the Developer perspective, navigate to the +Add page and select the From Catalog option to view all the available services in the Developer Catalog.
. Select Databases to list all the database services, and then click on the MariaDB option to see the details for the service.

Option 2:
. In the Developer perspective, navigate to the +Add page and select the Database option to view all the available database services present in the Developer Catalog under Database.
. Select MariaDB option to see the details for the service.


WDYT @Preeticp

Reply:
@gajanan-more option 1 seems perfect.

Reply:
Option 1 seems fine to me also. Will incorporate that in the new commit.